doc/man1/openssl-cmp.pod.in

```diff
@@ -1,49 +1,26 @@
 =pod
 {- OpenSSL::safe::output_do_not_edit_headers(); -}
 
+
```

Comment:
Extra empty line

Reply:
Yeah, so far before (sub-)section headers there were partly one, partly two empty lines.
At first in this PR I made this consistent in the direction of two empty lines,
but according to your comments I've just replaced all double empty lines by single ones.